covidWallet/app/components/SettingsScreen.js

```diff
@@ -1,13 +1,145 @@
+import React, { useState } from 'react'
+import { StyleSheet, Text, View, FlatList } from 'react-native'
+import { TextTypeView, BooleanTypeView } from './ShowTypesView'
 
-import * as React from 'react';
-import { View, Text } from 'react-native';
+var settingLocalData = {
+  GENERAL: {
```

Comment:
Please add changes that are discussed in the sprint review meeting. If there are any questions or confusion. Let's have a discussion.